src/Paket.Core/ScriptGeneration.fs

```diff
@@ -121,10 +123,11 @@ module ScriptGeneration =
       input.OrderedRelativeDllReferences
       |> Seq.map (sprintf """#r "%s" """)
 
-    depLines
-    |> fun lines -> Seq.append lines framworkRefLines
-    |> fun lines -> Seq.append lines dllLines
-    |> fun lines -> Seq.append lines [ sprintf "System.Console.WriteLine(\"Loaded {0}\", \"%s\");" packageName ]
+    let lines = Seq.concat [depLines; framworkRefLines; dllLines]
+
+    if Seq.isEmpty lines 
```

Comment:
Maybe we should change the type of the ScriptGenInput fields to `string list` instead of `string seq` to make it more obvious to the reader that it doesn't matter that we enumerate the sequence multiple times.